src/main/java/dev/ai4j/openai4j/image/GenerateImagesResponse.java

```diff
@@ -0,0 +1,122 @@
+package dev.ai4j.openai4j.image;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Represents the response from the OpenAI API when generating images.
+ *
+ * @param data A list of image data, each containing an image URL, Base64 encoded JSON response, and revised prompt
+ */
+public class GenerateImagesResponse {
+
+  private final List<ImageData> data;
+
+  public GenerateImagesResponse(Builder builder) {
+    this.data = builder.data;
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public List<ImageData> data() {
+    return data;
+  }
+
+  public static class ImageData {
+
+    private String url;
```

Comment:
minor: you can change `String` to `URI` here, gson knows how to deserialize it

Reply:
done